packages/yew-macro/src/lib.rs

```diff
@@ -150,3 +151,14 @@ pub fn hook(attr: TokenStream, item: TokenStream) -> proc_macro::TokenStream {
         .unwrap_or_else(|err| err.to_compile_error())
         .into()
 }
+
+/// Declares HTML element
+///
+/// Every HTML element is a component, which returns a manually constructed `VTag`.
+/// This macro is used to generate this component.
```

Comment:
This mean anyone can declare their custom components and use them directly in `html!`.
Could you please add docs for this?

Reply:
Yes, it means that but that macro is not for public usage (yet). There's missing functionality and I would like to make it usable from outside before documenting it. For example, custom elements expose events which are impossible to listen to using components generated by this macro.